app/products/calls/alerts.ts

```diff
@@ -125,3 +133,51 @@ const doJoinCall = async (serverUrl: string, channelId: string, isDMorGM: boolea
         setTimeout(() => unmuteMyself(), 1000);
     }
 };
+
+export const recordingAlert = (intl: IntlShape) => {
+    if (recordingAlertLock) {
+        return;
+    }
+    recordingAlertLock = true;
+
+    const {formatMessage} = intl;
+
+    const participantMessage = formatMessage({
+        id: 'mobile.calls_participant_rec',
+        defaultMessage: 'The host has started recording this meeting. By staying in the meeting you give consent to being recorded.',
+    });
+
+    Alert.alert(
```

Comment:
This alert should be made non-dissmisable (applies to android)

Reply:
It is non-dismissable by default.

Reply:
sorry @cpoile but not as far as I'm aware, and looking into the docs it does not seem to be the case

https://reactnative.dev/docs/next/alert#options

<img width="837" alt="image" src="https://user-images.githubusercontent.com/6757047/204735203-9ed9f199-40b6-4fcc-87d6-dcc0d0bcf783.png">

Reply:
No worries. It says `It is disabled by default` in the docs, and I've tested it.
https://reactnative.dev/docs/alert#android

Reply:
@enahum No worries. It says `It is disabled by default` in the docs, and I've tested it.
https://reactnative.dev/docs/alert#android